app/assets/javascripts/components/overview/statistics_update_info.jsx

```diff
@@ -49,37 +55,38 @@ const StatisticsUpdateInfo = createReactClass({
   render() {
     const course = this.props.course;
 
-    if ((Features.wikiEd && !course.ended) || !course.updates.last_update) {
+    if (Features.wikiEd && !course.ended) {
       return <div />;
     }
 
+    const updateTimesInformation = this.getUpdateTimesInformation();
     // Render Modal
     if (this.state.showModal) {
       return (
         <StatisticsUpdateModal
           course={course}
-          getUpdateTimesInformation={this.getUpdateTimesInformation}
+          updateTimesInformation={updateTimesInformation}
           toggleModal={this.toggleModal}
         />
       );
     }
 
-    const updateTimesInformation = this.getUpdateTimesInformation();
     let updateTimesMessage = '';
     if (updateTimesInformation !== null) {
```

Comment:
Under what circumstances could `updateTimesInformation` be `null`?

Reply:
I don't think it is possible. It wasn't my code and it was no problem for me, so I left it. I'll remove it and see what is happening